stdlib/bz2.pyi

```diff
@@ -88,7 +88,7 @@ def open(
     newline: str | None = ...,
 ) -> BZ2File | TextIO: ...
 
-class BZ2File(BaseStream, IO[bytes]):
+class BZ2File(BaseStream, IO[bytes]):  # type: ignore # argument disparities between base classes
```

Comment:
Agree with a comment above that we should make IO params pos-only, but this is fine for now.